config/config_models_test.tsv

```diff
@@ -1,3 +1,13 @@
-model	dataset	ABC_directory	feature_table	polynomial
-DNase_megamap_8features	DNase_megamap		resources/feature_tables/final_feature_set_DNase_hic.tsv	False 
-DNase_intact_8features	DNase_intact		resources/feature_tables/final_feature_set_DNase_hic.tsv	False
+model	dataset	ABC_directory	feature_table	polynomial	override_params
+DNase_megamap_noReg	DNase_megamap		resources/feature_tables/all_features_DNase_hic.tsv	False 	
+DNase_megamap_L1	DNase_megamap		resources/feature_tables/all_features_DNase_hic.tsv	False 	{'solver': 'saga', 'penalty': 'l1'}
```

Comment:
I realize this is important for us to recognize what are the best hyperparameters for our logistic regression model. Do we expect users to want to choose their own hyperparameters? Or are we going to find the best ones and make them the default?

Reply:
No idea who is going to use this besides us... the default parameters are already specified in config_training which is more transparent and explicit than having it hardcoded. The override_params can just be used if the user wants to change one of those for any particular model. The whole point of this workflow is to compare different model architectures so I feel like it;s a valid things to include!